_includes/payment-order-checkout-payments-only.md

```diff
@@ -165,7 +165,7 @@ Content-Type: application/json
 | {% icon check %} | └─➔&nbsp;`lastName`                    | `string`     | The last name of the payer.                                                                                                                                                                                                                                                                              |
 |                  | └─➔&nbsp;`email`                   | `string`     | The e-mail address of the payer. Will be used to prefill the Checkin as well as on the payer's profile, if not already set. Increases the chance for [frictionless 3-D Secure 2 flow]({{ features_url }}/core/3d-secure-2).                                                                             |
 |                  | └─➔&nbsp;`msisdn`                  | `string`     | The mobile phone number of the Payer. Will be prefilled on Checkin page and used on the payer's profile, if not already set. The mobile number must have a country code prefix and be 8 to 15 digits in length. The field is related to [3-D Secure 2]({{ features_url }}/core/3d-secure-2).            |
-|                  | └─➔&nbsp;`payerReference`                     | `string`     | A reference used in the Merchant Authenticated Consumer and Payments implementations to recognize the payer in the absence of SSN and/or a secure login. Read more about this in the [payerReference](/checkout-v3/payments-only/features/optional/mac-payer-reference) feature section.                                                                                                                                                                                                                                        |
+|                  | └─➔&nbsp;`payerReference`                     | `string`     | A reference used in the Enterprise and Payments implementations to recognize the payer in the absence of SSN and/or a secure login. Read more about this in the [payerReference](/checkout-v3/payments-only/features/optional/enterprise-payer-reference) feature section.                                                                                                                                                                                                                                        |
```

Comment:
I made some changes to this in the PayerReference task, but I guess we can solve that :)

Reply:
Yeah let's do that!